sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -76,18 +80,14 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
             string basicDtPayload = JsonSerializer.Serialize(basicTwin);
 
-            Response<string> createBasicDtResponse = await client.CreateDigitalTwinAsync(basicDtId, basicDtPayload);
-            Console.WriteLine($"Created digital twin with Id {basicDtId}. Response status: {createBasicDtResponse.GetRawResponse().Status}.");
-
-            #endregion Snippet:DigitalTwinsSampleCreateBasicTwin
+            Response<string> createBasicDtResponse = await client.CreateDigitalTwinAsync("basicDigitalTwin-123-Id", basicDtPayload);
```

Comment:
may be better to define these ids/strings within snippet and use that variable here and other places below.

Reply:
See the comment at the top of the file as to why we're using hard-coded strings throughout.